src/umpire/strategy/MonotonicAllocationStrategy.cpp

```diff
@@ -34,14 +34,23 @@ MonotonicAllocationStrategy::MonotonicAllocationStrategy(
   m_block = m_allocator->allocate(m_capacity);
 }
 
+MonotonicAllocationStrategy::~MonotonicAllocationStrategy()
```

Comment:
I don't think this function is reachable?

Reply:
Correct, none of the `AllocationStrategy` destructors get called anymore. However I thought I'd leave them in there so that no one had to re-implement them. However I'm happy to remove them all.

Reply:
That would be good. We'll keep your other branch around for future reference.

Reply:
Will do.